creator-node/src/routes/files.js

```diff
@@ -352,7 +352,6 @@ module.exports = function (app) {
 
       req.logger.debug('ipfs add resp', resizeResp)
     } catch (e) {
-      req.logger.error('image upload error', JSON.stringify(e))
```

Comment:
why remove this? its valuable to have these in the logs to be able to debug issues

Reply:
this log can definitely be useful! I just didn't think it was relevant to the scope of this PR and I introduced it in this PR

Reply:
ahhhh i didn't realize you had added it in an earlier commit, ok sounds good